runtime-watcher/main.go

```diff
@@ -43,7 +43,7 @@ var buildVersion = "not_provided"
 func main() {
 	var printVersion bool
 	flag.BoolVar(&printVersion, "version", false, "Prints the watcher version and exits")
-	logger := ctrl.Log.WithName("skr-webhook")
+	logger := ctrl.Log.WithName("skr-webhook").V(0)
```

Comment:
isn't this debugging level? do we really need this?

Reply:
I'd like to keep it for a bit until we close the metrics story. The log is anyways not too active usually.